docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,86 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+### `rewrite-table-path`
+
+This procedure rewrites an iceberg table's metadata files to a new location by replacing all source prefixes in absolute paths
+with a specified target prefix. After copying both metadata and data to the desired location, the replicated iceberg
+table will appear identical to the source table, including snapshot history, schema and partition specs.
+
+!!! info
+this procedure serves as the starting point to fully or incrementally copying an iceberg table to a new location. Copying all
+metadata and data files from source to target location is not included as part of this procedure.
+
+| Argument Name      | Required? | Type   | Description                                                                                                |
+|--------------------|-----------|--------|------------------------------------------------------------------------------------------------------------|
+| `table`            | ✔️        | string | Name of the table                                                                                          |
+| `source_prefix`    | ✔️        | string | source prefix to be replaced                                                                               |
+| `target_prefix`    | ✔️        | string | target prefix                                                                                          |
+| `start_version`    |           | string | First metadata version to rewrite, identified by name of a metadata.json file in the table's metadata log. |
+| `end_version`      |           | string | Last metadata version to rewrite, identified by name of a metadata.json file in the table's metadata log.  |
+| `staging_location` |           | string | Custom staging location                                                                                    |
```

Comment:
The location for newly modified metadata files, the output directory for this command(?)